packages/protocol/src/data/inscription.ts

```diff
@@ -55,6 +56,7 @@ export interface Inscription {
   schemaKey: SchemaKey;
   scriptingDataArgs: ScriptingDataArgs;
   string: string;
+  typeSearchRequest: TypeSearchRequest;
```

Comment:
This seems to be coming from another story. May lead to merge conflict with Elio. 
The faster of you two will be happy.